orchestra/static/orchestra/task/js/controllers.js

```diff
@@ -72,7 +72,12 @@
 
     vm.confirmSubmission = function(command, totalSeconds) {
       vm.submitting = true;
-      orchestraService.signals.fireSignal('submit.before');
+      if (orchestraService.signals.fireSignal('submit.before') === false) {
```

Comment:
style points---they just clicked submit and are at the bottom of the task, so they might think something broke because hte error is at the top.  throw a message or tooltip on the button saying something went wrong?

totally OK (and encouraged) for you to turn this into a `TODO(jrbotros)`